23/README.md

```diff
@@ -0,0 +1,283 @@
+
+## Methods
+<!--
+METHODS. The methods section will help you determine exactly how the authors performed the experiment.
+
+The methods describes both specific techniques and the overall experimental strategy used by the scientists. Generally, the methods section does not need to be read in detail. Refer to this section if you have a specific question about the experimental design.
+-->
+
+The performance tests were run on the Microsoft Azure cloud infrastructure. For the BigchainDB nodes, we choose compute optimized virtual machines from the [Fsv2 series][azure:fsv2], specifically *Standard F32s v2* (32 vCPUs, 64GiB memory, SSD drives with estimated IOPS limit of 5000). For the machine generating the workload, we used a *Standard F8s v2* (8 vCPUs, 16GiB memory). The virtual machines were all in the same datacenter.
+
+Note that the IOPS limit is crucial. A low number will create a bottleneck in disk reads and writes, slowing down database operations. To have a rough idea on the actual IO speed of the disk, `hdparam` was used.
+
+```
+benchmark1@benchmark1:~$ sudo hdparm -Tt /dev/sda
+
+/dev/sda:
+ Timing cached reads:   19514 MB in  1.99 seconds = 9785.28 MB/sec
+ Timing buffered disk reads: 1484 MB in  3.00 seconds = 494.39 MB/sec
+```
+
+All machines were provided with the operating system *Ubuntu 18.04.1 LTS (Bionic Beaver)*. For simplicity we provided all machines (both the machine generating the workload and the machines running the BigchainDB nodes) with the same software:
+
+- BigchainDB, [commit 6a9064196ad49baf3933051be6e116a3440fbad2][bdb:git-commit]
```

Comment:
Giving the specific Git commit is helpful for coders and anyone trying to replicate the results, but it's not helpful for the average reader. Maybe add something to give them a sense of where that commit fits into the BigchainDB version history, like 

> BigchainDB, commit blahblahblah (between version 2.0.0b5 and 2.0.0b6)